### ISSUE TITLE
Traceback, ranger  crash

### ISSUE THREAD

**Issue by thecsw** [ranger/ranger#1298]
#### Runtime Environment

- Operating system and version: Linux lennarc 4.18.5-arch1-1-ARCH #1 SMP PREEMPT Fri Aug 24 12:48:58 UTC 2018 x86_64 GNU/Linux
- Terminal emulator and version: urxvt
- Python version:  3.7.0 (default, Jul 15 2018, 10:44:58) [GCC 8.1.1 20180531]
- Ranger version/commit: ranger-master 1.9.1
- Locale: en_US.UTF-8

#### Context
I was just browsing through my hard drive files

#### Steps to reproduce
1. 
2. 
3. 

#### Traceback

```
ranger version: ranger-master 1.9.1
Python version: 3.7.0 (default, Jul 15 2018, 10:44:58) [GCC 8.1.1 20180531]
Locale: en_US.UTF-8
Current file: '/media/sdb1-usb-TOSHIBA_STOR.E_S/Backups/ntuser.ini'

Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/ranger/core/main.py", line 196, in main
    fm.loop()
  File "/usr/lib/python3.7/site-packages/ranger/core/fm.py", line 390, in loop
    ui.redraw()
  File "/usr/lib/python3.7/site-packages/ranger/gui/ui.py", line 338, in redraw
    self.draw()
  File "/usr/lib/python3.7/site-packages/ranger/gui/ui.py", line 365, in draw
    DisplayableContainer.draw(self)
  File "/usr/lib/python3.7/site-packages/ranger/gui/displayable.py", line 256, in draw
    displayable.draw()
  File "/usr/lib/python3.7/site-packages/ranger/gui/widgets/view_miller.py", line 100, in draw
    DisplayableContainer.draw(self)
  File "/usr/lib/python3.7/site-packages/ranger/gui/displayable.py", line 256, in draw
    displayable.draw()
  File "/usr/lib/python3.7/site-packages/ranger/gui/widgets/browsercolumn.py", line 176, in draw
    self._draw_file()
  File "/usr/lib/python3.7/site-packages/ranger/gui/widgets/browsercolumn.py", line 203, in _draw_file
    Pager.draw(self)
  File "/usr/lib/python3.7/site-packages/ranger/gui/widgets/pager.py", line 101, in draw
    self._draw_line(i, line)
  File "/usr/lib/python3.7/site-packages/ranger/gui/widgets/pager.py", line 132, in _draw_line
    self.addstr(chunk)
  File "/usr/lib/python3.7/site-packages/ranger/gui/curses_shortcuts.py", line 37, in addstr
    self.win.addstr(*args)
ValueError: embedded null character

ranger crashed. Please report this traceback at:
https://github.com/ranger/ranger/issues
```

**Comment by toonn** [comment]
Are you running ranger from the repos or from the AUR?

**Comment by thecsw** [comment]
Hey! I think I just installed it via `pacman`. I've been using ranger daily for almost 2 years now, this error is the first one!

**Comment by toonn** [comment]
Wow, that's a pretty good testimonial : ) This bug occurs because of a change in python 3.6 I believe. There's another one like it that came with python 3.7 something about StopIteration. The former was fixed in #1091 the latter in #1244.

You could install ranger-git from the AUR to get these fixes *right now* but we're planning a release that'll have them soon™.
